data/olm-catalog/ssp-operator.clusterserviceversion.yaml

```diff
@@ -500,27 +423,14 @@ spec:
           - virtualmachines/stop
           verbs:
           - update
-        - apiGroups:
```

Comment:
Did you enable all feature gates during testing? SSP can deploy Tekton resources.

Reply:
Yes, Tekton tasks have also been tested.